### ISSUE TITLE
[1.16.5] Possible incompatibility with the mod Scaling Health? 

### ISSUE THREAD

**Issue by antibang** [kwpugh/PowderPower#18]
I'm unsure about this and don't really know where else to ask, but I've noticed with the health tokens players **never** have to eat. I removed it from the belt slot and watched my food bar go down, but once I placed the token back in its slot the food bar instantly went up to full.

It doesn't seem to provide much health regeneration, if any,  but that's why I'm thinking maybe it's on Scaling Health's side, unless this is either an intended mechanic or that's just how regen works in general!

I've made the recipe for the token as difficult as I can while still being fair, and I know the player's of my modpack will not be happy with me if I make it even harder lol I'm using the most recent version for 1.16.5

**Comment by kwpugh** [comment]
I'll. take look.  it should give health regeneration not. food.

**Comment by kwpugh** [comment]
Looking at the code, I have it set to slowly increase player food and saturation, which will keep the player near or at full health hearts.

What was you expectation of the. item?

**Comment by kwpugh** [comment]
I have been playing a bit with scaling health and the health tokens.  It seems to perform as designed.   the health tokens restore hearts when lost.   I'm not sure what the issue is.

**Comment by antibang** [comment]
Oh, okay! I figured the token was working fine and thought that Scaling Health just had some weird incompatibility. I thought it would just have an effect like regeneration, but saturation makes sense as well. 

I'm sure there's a mod feature somewhere else in my modpack that is affecting the food issue. I've used your mod for a really long time and I don't remember the token ever completely preventing hunger. Though, could just be bad memory and possibly something else was added that also affects the hunger system in some way. No worries though. I just thought I'd ask so I could make a decision on the token's change :)
